src/typing/env_js.ml

```diff
@@ -44,7 +48,10 @@ let swap_changeset f =
   changeset := newset;
   oldset
 
-let global_scope: scope = ref SMap.empty
```

Comment:
Fair warning, the env/global_scope situation was tweaked recently (actually, it's the stuff @mroch did that makes it possible to refine globals). Shouldn't be too hard to merge, but nontrivial.